lua/nvim-tree/renderer/builder.lua

```diff
@@ -157,6 +151,15 @@ function Builder:_build_folder(node)
     foldername_hl = "NvimTreeEmptyFolderName"
   end
 
+  local foldername = get_folder_name(node) .. self.trailing_slash
```

Comment:
Any specific reason to move this block of code?

I'm probably missing something obvious...